src/adr

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
-eval "$($(dirname $0)/adr-config)"
+
+dir=$(dirname "$(readlink -f "$0")")
```

Comment:
As I mentioned on #53, `readlink -f` doesn't work on MacOS/BSD, so this bash-specific change should work:

```
dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
```

Reply:
Aside from the bash-specific issue, this will only work as long as the last component of the path used to find the script is not a symlink (according to https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within)